### PR TITLE
Change log v51.10.0

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Updates the changelog for v51.10.0 of the extensions.

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+- Update docs to reflect Apex library setting has been removed ([PR #3148](https://github.com/forcedotcom/salesforcedx-vscode/pull/3148))
+
+#### salesforcedx-vscode-apex
+
+- Update "All tests" option for `SFDX: Run Apex Tests` command to match CLI behavior ([PR #3126](https://github.com/forcedotcom/salesforcedx-vscode/pull/3126))
+
+#### salesforcedx-vscode-core
+
+- Fix issues with the following types for deploy/retrieve library ([PR #3147](https://github.com/forcedotcom/salesforcedx-vscode/pull/3147), [Issue #3114](https://github.com/forcedotcom/salesforcedx-vscode/issues/3114), [Issue #3157](https://github.com/forcedotcom/salesforcedx-vscode/issues/3157)):
+  - TimeSheetTemplate
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:16]: Would it be worth mentioning that these were also added? timesheettemplate and accountrelationshipsharerule

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md:16]: Ah yeah i forgot account relationship, thanks

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+## Fixed
+
+#### docs
+
+- Update docs to reflect Apex library setting has been removed ([PR #3148](https://github.com/forcedotcom/salesforcedx-vscode/pull/3148))
+
+#### salesforcedx-vscode-apex
+
+- Update "All tests" option for `SFDX: Run Apex Tests` command to match CLI behavior ([PR #3126](https://github.com/forcedotcom/salesforcedx-vscode/pull/3126))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]: Good catch - the change log generation didn't pick this one up correct?

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: It actually did I just reworded it :)

**Review by sfsholden — APPROVED** [review]

**Pull request merged**